src/entities/inline-fragments/mixins/ParentNode/index.ts

```diff
@@ -121,6 +121,10 @@ export function ParentNode<C extends { new(...args: any[]): InlineNode }>(constr
     public removeChild(child: ChildNode): void {
       const index = this.children.indexOf(child);
 
+      if (index === -1) {
+        return;
```

Comment:
Maybe throw error here?

Reply:
No, there is a recursion to support transparent tree operations, so this is an exit condition